plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -3467,4 +3467,26 @@ public void testTrinoCacheInvalidatedOnCreateTable()
 
         assertUpdate("DROP TABLE " + tableName);
     }
+
+    @Test
+    public void testQueriesWithoutCheckpointFiltering(){
```

Comment:
stupid question but why test only the case when checkpoint filtering is disabled?

Reply:
Because all the other tests have checkpoint filtering enabled ?